fabtests/Makefile.am

```diff
@@ -41,6 +41,7 @@ bin_PROGRAMS = \
 	functional/fi_rdm_stress \
 	functional/fi_multi_recv \
 	functional/fi_bw \
+	functional/fi_flood \
```

Comment:
This test is just adding a new mode to the bw test - I would just replace/rename the bw test and add the new testing mode inside. No need to create a whole new test

Reply:
FWIW, AWS CI has such flood_peer test that reuse fi_bw: https://github.com/ofiwg/libfabric/blob/main/fabtests/pytest/efa/test_flood_peer.py#L6